src/btest.cr

```diff
@@ -64,19 +61,19 @@ class Config
   end
 end
 
-
 # A collection of test cases and the list of runners which those test cases
 # should be run with.
 class Suite
-  def initialize(config : Config, path : String)
-    @config = config
+  getter config, path, name, cases, runners, templates
+
+  def initialize(@config : Config, path : String)
     @name = Suite.path_to_name(config, path)
 
     begin
-        file = YAML.parse(File.read(path))
+      file = YAML.parse(File.read(path))
```

Comment:
Use `File.open` instead of `File.read`, so the YAML parser can read directly from the file instead of reading the file first (allocating a (big?) string), then parsing the string.